src/Workspaces/Core/Portable/CodeActions/CodeAction_Cleanup.cs

```diff
@@ -89,7 +90,7 @@ internal static async Task<Solution> CleanSyntaxAndSemanticsAsync(
             using var _ = ArrayBuilder<(DocumentId documentId, CodeCleanupOptions options)>.GetInstance(documentIds.Length, out var documentIdsAndOptions);
             foreach (var documentId in documentIds)
             {
-                var document = changedSolution.GetRequiredDocument(documentId);
+                var document = await changedSolution.GetRequiredDocumentAsync(documentId, includeSourceGenerated: true, cancellationToken).ConfigureAwait(false);
```

Comment:
generally speaking, i would like us to have documentatino on calls that pass includeSourceGenerated with just a minor comment exaplining the scenario.

For example, here we might say: it may seem odd to be cleaning a generated doc.  However, taht's what happens in razor when we run a code fix on normal+razor code.  And razor will then take this change and map back into their space.

etc.

that way someone can read this and go: oh.. ok, i see why we need this.